devVerifiers.md

```diff
@@ -40,34 +40,46 @@ await CredentialHandlerPolyfill.loadOnce();
 console.log('Ready to work with credentials!');
 ```
 ## Construct a Web Credential Query
-A verifier can request credentials over CHAPI by forming a `VerifiablePresentationRequest` object, wrapped with additional information for working with *Web Credentials*.  The example below illustrates a `QueryByExample`, which is wrapped in a `web` object.  The `recommendedHandlerOrigins` parameter allows verifiers to suggest Credential Handlers (e.g. digital wallets) for the user to present the requested data. 
+#### 1. Make a Verifiable Presentation Request
+A verifier can request credentials over CHAPI by forming a *[Verifiable Presentation Request](https://w3c-ccg.github.io/vp-request-spec/)*.  The example below illustrates a `QueryByExample`, which specifies the type of Verifiable Credential being requested and an optional reason for the request.
 
 ```
-const credentialQuery = {
-  "web": {
-    "VerifiablePresentation": {
-            "query": [{
-                "type": "QueryByExample",
-                "credentialQuery": {
-                    "reason": "We need to see your University Degree to continue the teacher application process."
-                    "example": {
-                        "@context": [
-                            "https://w3id.org/credentials/v1",
-                            "https://www.w3.org/2018/credentials/examples/v1"
-                        ],
-                        "type": ["UniversityDegreeCredential"],
-                        "credentialSubject": {
-                            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
-                        }
-                    }
-                }
-            }]
-        },
-        "recommendedHandlerOrigins": [
-            "https://chapi-demo-wallet.digitalbazaar.com"
-        ]
+const testVpr = {
+  "query": [{
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your University Degree to continue the teacher application process.",
+      "example": {
+        "@context": [
+          "https://w3id.org/credentials/v1",
+          "https://www.w3.org/2018/credentials/examples/v1"
+        ],
+        "type": ["UniversityDegreeCredential"],
+        "credentialSubject": {
+          "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+        }
+      }
```

Comment:
You don't need to include the quotation marks for keys outside of json when constructing an object like this.

```suggestion
const testVpr = {
  query: [{
    type: "QueryByExample",
    credentialQuery: {
      reason: "Please present your University Degree to continue the teacher application process.",
      example: {
        "@context": [
          "https://w3id.org/credentials/v1",
          "https://www.w3.org/2018/credentials/examples/v1"
        ],
        type: ["UniversityDegreeCredential"],
        credentialSubject: {
          "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
        }
      }
```